JPS_Ontology/ontology/ontoems/OntoEMS.owl

```diff
@@ -835,7 +835,28 @@
         <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoems/</rdfs:isDefinedBy>
         <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Nitrogen oxides concentration</rdfs:label>
     </owl:Class>
-    
+
+
+    <!-- https://www.theworldavatar.com/kg/ontoems/CarbonDioxideConcentration -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontoems/CarbonDioxideConcentration">
+        <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontoems/AirPollutantConcentration"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The concentration of carbon dioxide in the ambient air.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoems/</rdfs:isDefinedBy>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Carbon dioxide concentration</rdfs:label>
+    </owl:Class>
+
+
+    <!-- https://www.theworldavatar.com/kg/ontoems/CarbonMonoxideConcentration -->
+
+
+    <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontoems/CarbonMonoxideConcentration">
+        <rdfs:subClassOf rdf:resource="https://www.theworldavatar.com/kg/ontoems/AirPollutantConcentration"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The concentration of carbon monoxide in the ambient air.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:datatype="http://www.w3.org/2001/XMLSchema#string">https://www.theworldavatar.com/kg/ontoems/</rdfs:isDefinedBy>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Carbon monoxide concentration</rdfs:label>
+    </owl:Class>
 
```

Comment:
I think you missed the definition of "MassOfSubstanceConcentration" as subClass of "Concentration".

![image](https://user-images.githubusercontent.com/79832739/201835384-84bd7c78-480a-436e-8849-aa152a0a3663.png)

Reply:
Furthermore, I believe we can simply use OM's "VolumeFraction" concepts for "VolumeOfSubstanceConcentration", see: https://raw.githubusercontent.com/cambridge-cares/OM/master/om-2.0.rdf

```
<owl:Class rdf:about="&om;VolumeFraction">
  <rdfs:label xml:lang="en">volume fraction</rdfs:label>
  <rdfs:label xml:lang="nl">volumefractie</rdfs:label>
  <om:alternativeLabel xml:lang="en">concentration (v/v)</om:alternativeLabel>
  <rdfs:subClassOf rdf:resource="&om;Quantity"/>
  <om:symbol>φ</om:symbol>
</owl:Class> 

<owl:Class rdf:about="&om;VolumeFraction">
  <om:commonlyHasUnit rdf:resource="&om;percent"/>
  <om:commonlyHasUnit rdf:resource="&om;partsPerMillion"/>
</owl:Class>
```